airflow/models.py

```diff
@@ -3524,9 +3543,34 @@ def run(
             donot_pickle=configuration.getboolean('core', 'donot_pickle'),
             ignore_task_deps=False,
             ignore_first_depends_on_past=False,
-            pool=None):
+            pool=None,
+            delay_on_limit_secs=1.0):
```

Comment:
Let's keep the default value in only one place (1.0), can this be a required arg instead (remove =1.0)?